mappings/net/minecraft/block/ChorusFlowerBlock.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/unmapped/C_knlkybdi net/minecraft/block/ChorusFlowerBlock
+	FIELD f_cviyldrt SUPPORT_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
```

Comment:
is this the moj? I think `STEM_SHAPE` or something would flow nicer but I don't have a strong opinion either way

Reply:
yes but moj is `SUPPORT_SHAPE`; I have went with it because the fence mappings used it; could consider `STEM_SHAPE` though!

Reply:
I like support shape then!

Reply:
wait frick i meant to reply with "moj is `BLOCK_SUPPORT_SHAPE`